src/components/ShowPoster.tsx

```diff
@@ -0,0 +1,51 @@
+import { ShowData } from '../types';
+import { Link } from 'react-router-dom';
+import { CardMedia } from '@mui/material';
+
+export const SHOW_POSTER_WIDTH = 180;
+
+export interface ShowPosterProps {
+    /**
+     * Movie or TV show metadata
+     */
+    details: ShowData;
+    /**
+     * Either 'movie' or 'tv'
+     */
+    showType: string;
```

Comment:
@Thenlie Would requiring `'movie' | 'tv'` here make more sense?

Reply:
Yeah it would. I will update this in my current work. Copied from show card so I will update that too